src/elements/common/theming/useCustomTheming.ts

```diff
@@ -11,15 +11,19 @@ const useCustomTheming = ({ selector, theme = {} }: ThemingProps) => {
         .join(';');
 
     useEffect(() => {
+        if (!styles) {
+            return undefined;
```

Comment:
We shouldn't return anything here for useEffect. We would want to just end execution with `return;` right?

Reply:
yes! I wanted to do that but there's an eslint rule about consistent returns: https://eslint.org/docs/latest/rules/consistent-return#treatundefinedasunspecified

basically `return;` (implicit undefined) doesn't work but `return undefined;` (explicit) does work 🤷

https://stackoverflow.com/questions/67658900/arrow-function-expected-no-return-value-with-clean-up-function-in-useeffect